### ISSUE TITLE
Control-Space shortcut no longer launches quicksilver on 1.6.0 

### ISSUE THREAD

**Issue by amancalledsun** [quicksilver/Quicksilver#2460]
After updating to 1.6.0 from 1.5.9 using the in-app updater, I can no longer start the Quicksilver shortcut using Control-Space.  The app shows as running in Activity Monitor but I cannot access it it anyway.  Force Quitting it and re-launching has no effect, similarly with rebooting and re-launching.  Reverting to 1.5.9 fixes this issue.  I'm running 10.10.5, on a 2010 Mac Mini with 16GB RAM.

**Comment by skurfer** [comment]
Nothing related to keyboard shortcuts or activation was changed in this version.

First, go to System Preferences → Users & Groups → You → Login Items and make sure “Hide” isn’t checked next to Quicksilver.

You could also try enabling the Dock icon temporarily and clicking it to see if it activates QS.

**Comment by amancalledsun** [comment]
Hi Rob, the Hide setting is/was not enabled in System Prefs. I am unable to enable the Dock icon because I am unable to even get to the launcher interface to get into Quicksilver preferences. Clearly something between 1.6.0 and MacOS 10.10.5 is not compatible.

**Comment by srsanborn** [comment]
I have a similar issue, though I might suggest this topic would be better written as "Quicksilver fails to fully launch".

I'm on macOS 10.11.6 on a Mac Pro (early 2008) with 14GB RAM. It appears both myself and @amancalledsun are on older versions of macOS, maybe it's an issue with legacy systems?

After updating from build 4024 to 4025 (using the in-app updater), I noticed a failure to launch the interface using ctrl-space.  Looking up the pid in Terminal and killing it is my only way to quit. So it appears to be (semi-)running, but I get no system menu icon (nor dock icon, though I have that disabled), nor response to the keyboard shortcut. Further evidence of the "running" state is an attempt to delete /Applications/Quicksilver.app via Finder fails with "can’t be moved to the Trash because it’s open".

The following log lines appear in Console when the 4025 build is re-launched after update, or launched from a stopped state (sans the "re-launch" line in the latter case):

9/24/18 10:30:14.913 PM Quicksilver[2422]: Relaunch from path /Applications/Quicksilver.app/Contents/MacOS/Quicksilver
9/24/18 10:30:19.452 PM Quicksilver[3547]: -[QSApp setAutomaticCustomizeTouchBarMenuItemEnabled:]: unrecognized selector sent to instance 0x7fdac0502ce0
9/24/18 10:30:19.452 PM Quicksilver[3547]: -[QSApp setAutomaticCustomizeTouchBarMenuItemEnabled:]: unrecognized selector sent to instance 0x7fdac0502ce0

Reverting to build 4024 restores functionality. No log lines appear in Console upon successful launch of this version.

**Comment by skurfer** [comment]
@srsanborn Thanks for finding those log messages. That’s definitely Touch Bar junk. I’ll try to prevent it from doing that. In the meantime, I’ve changed 1.6.0 to require 10.12 or higher.

**Comment by madranet** [comment]
Another pointless "me too" maybe. But just to confirm I'm getting exactly the same issue on OSX 10,11,6 and the same messages in the terminal:

```
$: /Applications/Quicksilver.app/Contents/MacOS/Quicksilver
2018-10-01 19:00:27.024 Quicksilver[1012:9907] -[QSApp setAutomaticCustomizeTouchBarMenuItemEnabled:]: unrecognized selector sent to instance 0x7fe42150b3c0
2018-10-01 19:00:27.024 Quicksilver[1012:9907] -[QSApp setAutomaticCustomizeTouchBarMenuItemEnabled:]: unrecognized selector sent to instance 0x7fe42150b3c0
```

Rolling back to 15,9 here too.

**Comment by skurfer** [comment]
I think Touch Bar support was added in 10.12, so that makes sense. It’ll be fixed in 1.6.1.
